engines/director/detection.cpp

```diff
@@ -275,10 +280,31 @@ ADDetectedGame DirectorMetaEngineDetection::fallbackDetect(const FileMap &allFil
 		desc->desc.filesDescriptions[0].fileName = s_fallbackFileNameBuffer;
 
 		Common::String extra;
+		Common::String sanitizedName;
 		Common::WinResources *exe = Common::WinResources::createFromEXE(&f);
 		if (exe) {
 			Common::WinResources::VersionInfo *versionInfo = exe->getVersionResource(1);
 			if (versionInfo) {
+				Common::String internalName = versionInfo->hash["InternalName"].encode();
+				Common::String fileDescription = versionInfo->hash["FileDescription"].encode();
+				if (!_fallback_blacklisted_names.contains(internalName)) {
+					if (extraInfo != nullptr) {
+						*extraInfo = new ADDetectedGameExtraInfo;
+						(*extraInfo)->gameName = internalName;
+
+						sanitizedName = ADGameDescription::sanitizeName(fileDescription.c_str(), fileDescription.size());
+						desc->desc.gameId = sanitizedName.c_str();
+					}
+				}
+				else if (!_fallback_blacklisted_names.contains(fileDescription)) {
```

Comment:
```suggestion
				} else if (!_fallback_blacklisted_names.contains(fileDescription)) {
```

Please follow our coding conventions